docs/en/status-communities/about-voting-to-change-community-visibility.md

```diff
@@ -0,0 +1,67 @@
+---
+id: 685
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# About voting to change community visibility
+
+All SNT holders can engage in [voting to include or exclude a community][vote-to make-a-community-private-or-public] from the Communities Discover page, and [voting to feature a community][vote-to-feature-a-community]. 
+
+Votes happen on the [community directory curation dApp][curate.status.app]. This is an independent, decentralized application deployed on the Optimism network and is separate from the Status app. Voting outcomes are solely determined by the voters, and Status has no control over these results.
+
+When a community is created, it's private by default, which is ideal for small, exclusive communities. However, if the community targets a broad audience, you might prefer to make it public.
+
+## The basics
+
+- Every SNT holder, not just community members or Status users, can participate in all voting steps.
+- Your voting power is determined by how many SNT tokens you stake.
+- You don't spend the tokens to vote, but you must hold the tokens during the validation and finalization periods, or your vote is discarded.
+- You need to pay a [gas fee][understand-network-fees] in ETH for initiating, validating and finalizing a vote. The amount of gas fee depends on the traffic of Optimism.
```

Comment:
Maybe it is worth to mention that most users don't need to pay gas, the voting itself is free.